packages/expo-modules-autolinking/src/reactNativeConfig/__tests__/reactNativeConfig-test.ts

```diff
@@ -9,14 +9,39 @@ import {
   findDependencyRootsAsync,
   resolveAppProjectConfigAsync,
   resolveDependencyConfigAsync,
+  findEdgeToEdgeDependencyRoot,
 } from '../reactNativeConfig';
 import type {
   RNConfigReactNativeLibraryConfig,
   RNConfigReactNativeProjectConfig,
 } from '../reactNativeConfig.types';
 
 jest.mock('fs/promises');
-jest.mock('resolve-from');
+jest.mock('resolve-from', () => ({
+  ...jest.requireActual('resolve-from'),
+  silent: (fromDirectory, request) => {
```

Comment:
any reason to have the custom mock for resolve-from here?

we have similar mock at https://github.com/expo/expo/blob/32ceab8e8f23a5b7439e55a223330c66fd308de0/packages/expo-modules-autolinking/__mocks__/resolve-from.ts
it should work by `jest.mock('resolve-from');`

Reply:
You are right I didn't know that it would pull the mock from there. Although the existing mock doesn't behave like the actual `resolve-from`.
The difference is that when you resolve a file it returns the full path instead of the directory, so when the new tests resolve a second time based on the prevoius results you get a path like
'app/package.json/node_modules/react-navtive-edge-to-egde/package.json'
I added two small changes to the mock that don't break the existing tests, and allow mine to pass